spyderlib/spyder_restart.py

```diff
@@ -0,0 +1,123 @@
+# -*- coding: utf-8 -*-

```

Comment:
I think I have the perfect suggestion: `restart_app.py`, given that we already have `start_app.py`.

IMHO, that will convey its meaning much better than the current name